repos/fdbt-site/src/pages/selectTimeRestrictions.tsx

```diff
@@ -17,16 +23,19 @@ interface SelectTimeRestrictionsProps {
     errors: ErrorInfo[];
     timeRestrictions: PremadeTimeRestriction[];
     selectedId: number | null;
+    backHref: string;
 }
 
 const SelectTimeRestrictions = ({
     csrfToken,
     errors,
     timeRestrictions,
     selectedId,
+    backHref,
 }: SelectTimeRestrictionsProps): ReactElement => {
     return (
         <FullColumnLayout title={title} description={description} errors={errors}>
+            {!!backHref && errors.length === 0 ? <BackButton href={backHref}></BackButton> : null}
```

Comment:
Can we change all of the backbuttons to be   <BackButton href={backHref} /> (self closing)

Reply:
ok ... will do that